LuaMenu/widgets/gui_maplist_panel.lua

```diff
@@ -176,7 +155,14 @@ local function InitializeControls()
 	--local lmkb, lmalloc, lgkb, lgalloc = Spring.GetLuaMemUsage()
 	--Spring.Echo("LuaMenu KB", lmkb, "allocs", lmalloc, "Lua global KB", lgkb, "allocs", lgalloc)
 
-	local Configuration = WG.Chobby.Configuration
+	wgChobbyConfig = WG.Chobby.Configuration
+
+	-- Sharing font saves about 20%, but getting the config for the TextBox font isn't straightforward.
+	local fontSpec = wgChobbyConfig:GetFont(2)
+	local sacrificialTextBox = TextBox:New({})
+	fontSpec.font = sacrificialTextBox.font.font
+	sacrificialTextBox:Dispose()
+	listFont = Font:New(fontSpec)
```

Comment:
Ugh. I couldn't work out a better way to get the text box font, but this was the biggest single performance boost I found.

Reply:
I'm pretty sure the fonts are cached somewhere in chili.

Reply:
They are, in fonthandler.

Without objectOverrideFont: 0.5374794 cycles
With objectOverrideFont: 0.45657158 cycles

I'll do some more digging to understand the difference when I have time. I suspect I'll find it's due to object init, again.

Reply:
Yeah it looks like the time is in Font:New and calling SetParent on hundreds of Font references.

The way chili styles are applied makes it impossible to get the text box font style another way (without just copy pasta). I'll back this out then. I'll consider messing around with a virtualized list in future work.